src/bcm/run/extract_discs_cords_sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+# Check for -h option
+if [ "$1" = "-h" ]; then
+    echo "This script is designed to compute disc labeling technics on an input dataset "
+    echo "The data is expected to be gathered using the src/bcm/utils/gather_data.py script. "
+    echo "The main steps of this script include: "
+    echo ""
+    echo "1. Creating and initializing a text file were all the coordinates for each method"
+    echo " where all the coordinates of the discs will be gathered "
+    echo "2. Running the scripts of the wanted methods available in this benchmark "
+    echo "3. Computing the metrics and creating the graphs and images to compare the different "
+    echo "approaches"
+    echo ""
+    echo "The script takes several command-line arguments for customization:"
+    echo "--datapath: path to the input dataset"
+
+    echo "Data organization - using the script gather_data.py:"
+    echo " data"
+    echo " ├── sub-errsm37"
+    echo " │   ├── sub-errsm37_T2w_labels-disc-manual.nii.gz"
+    echo " │   ├── sub-errsm37_T2w.nii.gz"
+    echo " │   ├── sub-errsm37_T1w_labels-disc-manual.nii.gz"
+    echo " │   └── sub-errsm37_T1w.nii.gz"
+    echo " └── sub-errsm38"
+    echo "     ├── sub-errsm38_T1w_seg-manual.nii.gz"
+    echo "     └── sub-errsm38_T1w.nii.gz"
+
+    exit 0
+fi
+
+# BASH SETTINGS
+# ======================================================================================================================
+
+# Uncomment for full verbose
+# set -v
+
+# Immediately exit if error
+set -e
+
+# Exit if user presses CTRL+C (Linux) or CMD+C (OSX)
+trap "echo Caught Keyboard Interrupt within script. Exiting now.; exit" INT
+
+# GET PARAMS
+# ======================================================================================================================
+
+# SET DEFAULT VALUES FOR PARAMETERS.
+# ----------------------------------------------------------------------------------------------------------------------
+# General parameters
+DATA_DIR=""
+CONTRAST=""
+OUTPUT_TXT=""
+SUFFIX_IMG=""
+SUFFIX_LABEL_DISC="_labels-disc-manual"
+SUFFIX_SEG="_seg"
+VERBOSE=0
+
+# Hourglass parameters
+SKELETON_DIR="../disc-labeling-hourglass/src/dlh/skeletons"
```

Comment:
hard-coded path-- indicate in the doc that the script needs to be run at a specific location